network-api/networkapi/templates/fragments/buyersguide/pni_sort_dropdown.html

```diff
@@ -29,10 +29,10 @@
     </li>
 
     
-    <li aria-expanded="false" role="list" class="tw-absolute tw-top-0 tw-hidden tw-left-0 pni-creepiness__list-container tw-w-[250px] tw-z-50 tw-bg-white tw-border tw-border-gray-20">
-      <ul class="pni-creepiness__list tw-w-full tw-list-none tw-m-0 tw-p-0">
+    <li aria-expanded="false" role="list" class="pni-creepiness__list-container tw-absolute tw-top-0 tw-invisible tw-select-none tw-pointer-events-none tw-right-0 tw-z-50 tw-bg-white tw-border tw-border-gray-20">
+      <ul class="pni-creepiness__list tw-w-max tw-list-none tw-m-0 tw-p-0">
```

Comment:
```suggestion
      <ul class="pni-creepiness__list tw-list-none tw-m-0 tw-p-0">
```

Hey Simon, I was finding that the dropdown covered a little bit of the "sort by" text:

![Screenshot 2023-01-22 at 18-42-38 Privacy Not Included A Buyer’s Guide for Connected Products](https://user-images.githubusercontent.com/18314510/213957802-c8e7cbb2-b4d4-4c12-a7c4-336ac985d63d.png)


However, I found that removing the class `tw-w-full` seems to fix this issue:


![Screenshot 2023-01-22 at 18-43-50 Privacy Not Included A Buyer’s Guide for Connected Products](https://user-images.githubusercontent.com/18314510/213957843-20fe39e3-a255-4fbd-a5bd-ed1d0eed8da2.png)